src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Controllers/PolicyController.cs

```diff
@@ -120,5 +123,47 @@ public async Task<ActionResult> GetRolesWithAccess(string org, string app)
 
             return Ok(PolicyHelper.GetRolesWithAccess(policy));
         }
+
+        /// <summary>
+        /// Gets a list of resource policies for the list of org/apps
+        /// </summary>
+        /// <param name="orgAppList">The list of org/apps</param>
+        /// <returns>A list resourcePolicyResponses</returns>
+        [AllowAnonymous]
+        [HttpPost("/authorization/api/v1/resourcepolicies")]
+        public async Task<ActionResult> GetResourcePolicies(List<List<AttributeMatch>> orgAppList)
```

Comment:
Did you consider getting the required identificators as query parameters? This would allow us to have this as a GET action rather than a POST.

Reply:
I agree. Unless I'm missing (forgetting?) something, this particular endpoint will only every require a list of apps as input, so I therefore don't see much point to being able to express arbitrary attribute match lists with `List<List<AttributeMatch>>`. This could probably just take a string of the form `org1/app1,org2/app2,org3/app3` which will be simpler to both produce and consume.

Reply:
Oh, I remember :) We want this to be generic, eg. make it possible to query what actions are delegable for a specific task/event within one or more apps, which will be cumbersome to implement as GET-parameters. The implementation as-is only handles org/app, but this can be changed later without breaking the API.

Since this is internal JSON-RPC not aspiring to be particulary RESTful, using POST makes the most sense.